README.md

```diff
@@ -1,2 +1,39 @@
-# sass-bundle
-Delightful Sass Support for Symfony + AssetMapper
+# Sass For Symfony!
+
+This bundle make it easy to use Sass with Symfony's AssetMapper Component
+(no Node required!).
+
+- Automatically downloads the correct Sass binary
+- Adds a `sass:build` command to build and watch your Sass changes
+
+## Installation
+
+Install the bundle
+
+```shell
+composer require symfonycasts/sass-bundle
+```
+
+## Usage
+
+To use generated CSS, start by including the input file
+(`assets/styles/app.css` by default)
```

Comment:
We should first tell them to create an `assets/styles/app.scss` file - even maybe give some very basic example file - e.g. one that sets a variable, then uses that to set the `background-color` of the `body` element.

Then it will make more sense whey we're pointing to `styles/app.scss` below. Also, type on the line above - it says `app.css` :)